internal/ccsmp/ccsmp_core.go

```diff
@@ -348,12 +348,12 @@ func (session *SolClientSession) solClientSessionSubscribeWithFlags(topic string
 	return handleCcsmpError(func() SolClientReturnCode {
 		cString := C.CString(topic)
 		defer C.free(unsafe.Pointer(cString))
-		// This is not an unsafe usage of unsafe.Pointer as we are using correlationId as data, not as a pointer
+		// This is not an unsafe usage of unsafe.Pointer as we are using dispatchID and correlationID as data, not as pointers
```

Comment:
looks like the comment was wrong before, we were not using unsafe.Pointer for correlationId before and we still are not.  So I don't think we need a comment here at all.